### PR TITLE
Inlined variables

### PR DESCRIPTION
**Pull request by SeanFeldman**
Connects to #341 

- Inlining variables

### CODE REVIEW

**Comment by nemakam** [comment]
Let me trigger a new build after merging with #347

File: src/Microsoft.Azure.ServiceBus/Core/MessageReceiver.cs [hunk 1]
```diff
@@ -1455,19 +1453,15 @@ Rejected GetRejectedOutcome(IDictionary<string, object> propertiesToModify, stri
                     rejected.Error.Info.Add(Message.DeadLetterErrorDescriptionHeader, deadLetterErrorDescription);
                 }
 
-                if (propertiesToModify != null)
+                foreach (var pair in propertiesToModify)
```
Comment (nemakam) [inline src/Microsoft.Azure.ServiceBus/Core/MessageReceiver.cs]: This has to be wrapped inside `if (propertiesToModify != null)`

Comment (SeanFeldman) [reply src/Microsoft.Azure.ServiceBus/Core/MessageReceiver.cs]: Silly mistake, sorry. Fixing.

**Comment by SeanFeldman** [comment]
@nemakam done.

**Pull request merged**